### ISSUE TITLE
ARIMA: plot_diagnostics does not show the white noise of standardized residual for "u"

### ISSUE THREAD

**Issue by FrankManc** [statsmodels/statsmodels#8750]
I am trying to use the function plot_diagnostics. My code is as following:
<img width="744" alt="image" src="https://user-images.githubusercontent.com/104614955/226793039-48d73e3b-3d83-45b1-8450-81cbf2905402.png">
The code is a simple ARIMA model. But, it does not return the plot of standardized residual for "u", everything else is expected.
The result is as following:
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/104614955/226793289-694e58d5-3595-48ce-9d47-8a215fe99a5c.png">

I have no idea what happened in my code, if anyone could help me?

By the way, I notice that x-axis is integer number, it is supposed to be datetime. ALSO, I already check the data type of my x-axis, which is datetime type.

**Comment by ChadFulton** [comment]
Thanks for reporting this. Would it be possible for you to provide a reproducible example?  The data values themselves can all be simulated, or set to zero, if that makes it easier to share the example.

**Comment by FrankManc** [comment]
> Thanks for reporting this. Would it be possible for you to provide a reproducible example? The data values themselves can all be simulated, or set to zero, if that makes it easier to share the example.

Sure, thank your replying. I will upload the example. 

import pandas as pd
import numpy as np
from statsmodels.tsa.arima.model import ARIMA
from sklearn.metrics import mean_absolute_percentage_error
import pandas as pd
import yfinance as yf
import matplotlib.pyplot as plt
import nasdaqdatalink
from statsmodels.tsa.seasonal import seasonal_decompose
import seaborn as sns
sns.set_theme(context="talk", style="whitegrid", 
              palette="colorblind", color_codes=True, 
              rc={"figure.figsize": [12, 8]})
%matplotlib inline
%config InlineBackend.figure_format = "retina"
nasdaqdatalink.ApiConfig.api_key = "Your Key"

df = (
    nasdaqdatalink.get(dataset="FRED/UNRATENSA",
                       start_date="2010-01-01",
                       end_date="2019-12-31")
    .rename(columns={"Value": "unemp_rate"})
)

TEST_LENGTH = 12
df_train = df.iloc[:-TEST_LENGTH]
df_test = df.iloc[-TEST_LENGTH:]

arima_212 = ARIMA(
    df_train["unemp_rate_log"], order=(2, 1, 2)
).fit()
arima_212.summary()

arima_212.plot_diagnostics(figsize=(18, 14), lags=25)

sns.despine()
plt.tight_layout()

This is my code. Thank you.

**Comment by ChadFulton** [comment]
I had to modify your code slightly (to add the "unemp_rate_log" variable), but wasn't able to replicate the figure you plotted above.

To make sure nothing else is going on in your code that isn't showing up in your example, would you try running the following code to confirm that you get the same figure as in your original post?

```python
import pandas as pd
import numpy as np
from statsmodels.tsa.arima.model import ARIMA
import pandas as pd
import matplotlib.pyplot as plt
import nasdaqdatalink
from statsmodels.tsa.seasonal import seasonal_decompose
import seaborn as sns
sns.set_theme(context="talk", style="whitegrid", 
              palette="colorblind", color_codes=True, 
              rc={"figure.figsize": [12, 8]})

df = (
    nasdaqdatalink.get(dataset="FRED/UNRATENSA",
                       start_date="2010-01-01",
                       end_date="2019-12-31")
    .rename(columns={"Value": "unemp_rate"})
)

df['unemp_rate_log'] = np.log(df['unemp_rate'])

TEST_LENGTH = 12
df_train = df.iloc[:-TEST_LENGTH]
df_test = df.iloc[-TEST_LENGTH:]

arima_212 = ARIMA(
    df_train["unemp_rate_log"], order=(2, 1, 2)
).fit()
arima_212.summary()

arima_212.plot_diagnostics(figsize=(18, 14), lags=25)

sns.despine()
plt.tight_layout()
```

Thanks

**Comment by FrankManc** [comment]
Thank you for your reply. The modification you made is correct. I may copy and paste my original code wrong just by accident. I just run the modification code, which returns the same plots as the uploaded plots.

Here is the screenshot of the modification code
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/104614955/227098395-6d3e8817-163d-43d8-9e0b-2c03e5012863.png">

**Comment by FrankManc** [comment]
Yes.

import statsmodels.api as sm
print(sm.__version__)


>>>>> 0.13.5

import pandas as pd
print(pd.show_versions())

>>>>> INSTALLED VERSIONS
------------------
commit           : bb1f651536508cdfef8550f93ace7849b00046ee
python           : 3.8.16.final.0
python-bits      : 64
OS               : Darwin
OS-release       : 22.3.0
Version          : Darwin Kernel Version 22.3.0: Mon Jan 30 20:42:11 PST 2023; root:xnu-8792.81.3~2/RELEASE_X86_64
machine          : x86_64
processor        : i386
byteorder        : little
LC_ALL           : None
LANG             : zh_CN.UTF-8
LOCALE           : zh_CN.UTF-8

pandas           : 1.4.0
numpy            : 1.21.6
pytz             : 2022.7.1
dateutil         : 2.8.2
pip              : 23.0.1
setuptools       : 67.4.0
Cython           : 0.29.33
pytest           : 7.2.1
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : 4.9.2
html5lib         : 1.1
pymysql          : None
psycopg2         : None
jinja2           : 3.1.2
IPython          : 8.10.0
pandas_datareader: None
bs4              : 4.11.1
bottleneck       : 1.3.6
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : 3.7.0
numba            : 0.56.4
numexpr          : 2.8.3
odfpy            : None
openpyxl         : 3.1.1
pandas_gbq       : None
pyarrow          : None
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.7.3
sqlalchemy       : 2.0.4
tables           : None
tabulate         : None
xarray           : 2023.1.0
xlrd             : 2.0.1
xlwt             : None
zstandard        : None
None

**Comment by bashtage** [comment]
I can't reproduce on main:

![image](https://user-images.githubusercontent.com/5585221/231175176-08f2a05e-b573-4b22-b786-01852d3629a0.png)
